oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/extractor/ExtractorSpec.java

```diff
@@ -269,13 +274,12 @@ public void slowSql(@DelegatesTo(SlowSqlSpec.class) final Closure<?> cl) {
         DatabaseSlowStatementBuilder builder = new DatabaseSlowStatementBuilder(namingControl);
         builder.setLayer(Layer.nameOf(log.getLayer()));
 
-        LocalDateTime localDateTime = Instant.ofEpochSecond(log.getTimestamp()).atZone(ZoneId.systemDefault()).toLocalDateTime();
-        String timeBucket = DTF.format(localDateTime);
-        builder.setTimeBucket(Long.parseLong(timeBucket));
+        long timeBucket = TimeBucket.getTimeBucket(log.getTimestamp() * 1000, DownSampling.Minute);
```

Comment:
Can we be sure, the timestamp is in second?

Reply:
No, but for more time user will use lua(os.time() to get current timestamp) in fluent-bit. So I use it.
I also feel that this way maybe not good.

Reply:
In the log definition, the report format is a timestamp. ref 

https://github.com/apache/skywalking-data-collect-protocol/blob/c2c9bbbff43f0ad9f917a0a55285538a6b45739e/logging/Logging.proto#L44

So, from my understanding, you should move this `times 1000` into the fluentbit side.

Ref `RecordSinkListener#L79`

Reply:
I understand.